src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/Application.vue

```diff
@@ -202,3 +207,9 @@ export default defineComponent({
   },
 });
 </script>
+
+<style scoped>
+.v-stepper-item--selected {
```

Comment:
I think we have a stepper scss file? Or if not do you mind adding one? Probably best to have the vuetify modifications all in one place.

Reply:
Great way to do it!